GOVERNANCE.md

```diff
@@ -0,0 +1,81 @@
+# Abstract
+
+The purpose of this document is to formalize the governance process used by the
+tskit-dev community for core software projects, to clarify who has what responsibilities,
+and how decisions are made on changes to those responsibilities.
+
+tskit-dev is an open and inclusive community. Anyone with an interest in the
+software and science can contribute discussion, and propose changes to code, documentation
+or website content.
+
+# Scope
+
+The governance model described in this document applies to the core
+repositories of the tskit software ecosystem. These are:
+
+- [tskit - C and Python API](http://github.com/tskit-dev/tskit)
+- [msprime - coalescent simulator](http://github.com/tskit-dev/msprime)
+- [tszip - compression library for tskit](http://github.com/tskit-dev/tszip)
+- [tskit-site - website for the ecosystem](http://github.com/tskit-dev/tskit-site)
+- [kastore - key-value store](http://github.com/tskit-dev/kastore)
+- [administrative](http://github.com/tskit-dev/administrative)
+- [.github - common config](http://github.com/tskit-dev/.github)
+
+
+# Roles
+
+## Contributors
+
+Anyone can become a contributor to the tskit-dev project by, for example,
```

Comment:
This sounds like a definition of "contributor" which might be interpreted in various concrete ways (eg having the rights to be on the slack); how about:
```suggestion
Anyone is welcome to contribute to the tskit-dev project by, for example,
```
To be clear, I think the default should be that anyone who contributes should probably be on the slack, but there are situations where they wouldn't be.